Userland/Libraries/LibGUI/TextDocument.cpp

```diff
@@ -170,10 +170,10 @@ TextDocumentLine::TextDocumentLine(TextDocument& document, StringView text)
     set_text(document, text);
 }
 
-void TextDocumentLine::clear(TextDocument& document)
+void TextDocumentLine::clear(TextDocument&)
 {
     m_text.clear();
-    document.update_views({});
+    // No need to call notify_did_change since all of our callers do that anyways.
```

Comment:
is this a public api? maybe it would be better to invoke update optionally as in other places? 🤔

Reply:
TextDocumentLine is an internal API so I think it is fine to rely on the callers doing the update callbacks. That's why I added the comment.